Source/UI/NewUI/BillingForm/ViewModel/DefaultBillingFormViewModel.swift

```diff
@@ -59,58 +59,90 @@ final class DefaultBillingFormViewModel: BillingFormViewModel {
         editDelegate = view
         return view
     }
-
-    func getCell(tableView: UITableView, indexPath: IndexPath,  delegate: CellTextFieldDelegate?) -> UITableViewCell {
-        guard style.cells.count > indexPath.row, var viewStyle = style.cells[indexPath.row].style
-        else { return UITableViewCell() }
-
-        updateStyle(with: &viewStyle, cellStyle: style.cells[indexPath.row], tag: indexPath.row)
-
-        return getCell(tableView: tableView,
-                       viewStyle: &viewStyle,
-                       cellStyle: style.cells[indexPath.row],
-                       indexPath: indexPath,
-                       delegate: delegate)
+    
+    func getCell(tableView: UITableView, indexPath: IndexPath, sender: UIViewController?) -> UITableViewCell {
+        guard style.cells.count > indexPath.row else { return UITableViewCell() }
+        
+        if style.cells[indexPath.row].index == BillingFormCell.country(nil).index {
```

Comment:
What does the nil mean here ? `BillingFormCell.country(nil).index`

Reply:
Also should we check if style.cells[indexPath.row] it not nil before using this, just to be safe.

Reply:
I cleaned this part to be more readable.
 
```guard style.cells.count > indexPath.row else { return UITableViewCell() }```
This line is preventing row to be greater than or equal to cell count.